doc/guides/getting_started/src/unofficial/fedora.md

```diff
@@ -0,0 +1,19 @@
+# Unofficial Setup Guide for Fedora
+
+This is an unofficial guide detailing how to set up a Fedora system for OpenTitan development.
+
```

Comment:
May want to add a sentence here saying that the only step that differs from the main guide (linked) is step 2.

Reply:
Done.